Lib/test/test_os.py

```diff
@@ -3122,6 +3122,23 @@ def cleanup():
             print("File:", filename)
             print("stat with access:", stat1)
 
+        # We should now not be able to open the file.
+        # If we can, the test isn't going to be useful.
+        try:
+            _winapi.CloseHandle(_winapi.CreateFile(
```

Comment:
Is it different from using `open()` or `os.open()`?

Reply:
Yeah, admin may have additional privileges that make `FILE_FLAG_BACKUP_SEMANTICS` able to access files that are supposed to be locked away. (One earlier issue with this test is that `CreateJunction` was enabling the privilege but not disabling it.)

The flags passed here match what `stat` uses, so we should get equivalent behaviour. Though there's a new quirk that I'm looking into, so we may end up having to just remove the checks in the test that make sure ino/dev are zero, since in some cases they may legitimately get the real value.